dev/arktype.io/docs/api/intersection.md

```diff
@@ -4,8 +4,24 @@ hide_table_of_contents: true
 
 # intersection
 
-## text
+## operator
 
-```ts
-intersection: Ark["intersection"]
-```
+-   [intersection](./intersection.md)
+
+## tuple
+
+-   [a, &, b]
```

Comment:
Let's go with `L` and `R` as an alternative to `T` when there are two operands. Also & needs to be quoted.